README-mux.md

```diff
@@ -0,0 +1,345 @@
+                        "pipeline_metadata": {
+                            "collector": {
+                                "inputname": "fluent-plugin-systemd",
+                                "ipaddr4": "10.128.0.26",
+                                "ipaddr6": "fe80::30e3:7cff:fe55:4134",
+                                "name": "fluentd openshift",
+                                "received_at": "2016-12-01T14:09:53.848788-05:00",
+                                "version": "0.12.29 1.4.0"
+                            }
+                        }
+                    },
+                    "_type": "com.redhat.viaq.common"
+                }
+            ],
+            "max_score": 1.0,
+            "total": 1453
+        },
+        "timed_out": false,
+        "took": 15
+    }
+
+## Getting the shared_key and CA cert
+
+In order to configure the client side of Fluentd secure_forward, you will need
+the values for the `shared_key` and the `ca_cert_path`.  For the first
+iteration, you will just use the ones generated by the `setup-mux.sh` script.
+
+    # oc project logging
+    # oc get secret logging-mux --template='{{index .data "mux-ca"}}' | base64 -d > mux-ca.crt
+    # openssl x509 -in mux-ca.crt -text | more
+    Certificate:
+    ...
+        Subject: CN=openshift-signer@1480618407
+    ...
+    # oc get secret logging-mux --template='{{index .data "mux-shared-key"}}' | \
+      base64 -d > mux-shared-key
+
+Use the `mux-ca.crt` and `mux-shared-key` to configure the Fluent
+secure_forward clients.  The `setup-mux.sh` script will generate these for you,
+but use the above instructions if you need to regenerate them.
+
+## Client side setup
+
+The client side setup should look something like this:
+
+    <match **something**>
+      @type secure_forward
+      self_hostname forwarding-${hostname}
+      ca_cert_path /path/to/mux-ca.crt
+      secure yes
+      enable_strict_verification true
+      shared_key "#{File.open('/path/to/mux-shared-key') do |f| f.readline end.rstrip}"
+      <server>
+        host mux.logging.test
+        hostlabel mux.logging.test
+        port 24284
+      </server>
+    </match>
+
+## Running Kibana
+
+You will first need to create an OpenShift user and assign this user rights to
+view the application and operations logs.  The install above uses the
+AllowAllPasswordIdentityProvider which makes it easy to create test users like
+this:
+
+    # oc project logging
+    # oc login --username=kibtest --password=kibtest
+    # oc login --username=system:admin
+    # oadm policy add-cluster-role-to-user cluster-admin kibtest
+
+Now you can use the `kibtest` username and password to access Kibana.  Just
+point your web browser at `https://kibana.logging.test` where the
+`logging.test` part is whatever you specified in the 
+`openshift_master_default_subdomain` parameter in the `vars.yaml` file.
```

Comment:
"`logging.test` part is whatever you specified in the 
 +`openshift_master_default_subdomain` parameter in the `vars.yaml` file."
This comment is very important before this part. This should be in line 140.